.gitlint

```diff
@@ -0,0 +1,17 @@
+# Commit msg lint config
+
+## https://jorisroovers.com/gitlint/latest/ignoring_commits/
+## https://jorisroovers.com/gitlint/latest/rules/builtin_rules/
+
+[general]
+regex-style-search=true
+ignore=B1,B6
+
+# Ensure every title starts with a user-story identifier like US123
+[title-match-regex]
+regex="feat(:|\(.*\):)|fix(:|\(.*\):)|style(:|\(.*\):)|refactor(:|\(.*\):)|test(:|\(.*\):)|docs(:|\(.*\):)|chore(:|\(.*\):)|build(:|\(.*\):)|ci(:|\(.*\):)|perf(:|\(.*\):)|metric(:|\(.*\):)|misc(:|\(.*\):)|release:"gi
+
+# Ignore Release commits
+[ignore-by-title]
+regex=Release(.*)
```

Comment:
I wonder if title refers to PR title or commit title, because our commit titles are also fine. 

Example https://github.com/lacework/go-sdk/pull/1282 

PR Title: `Release v1.21.0`
Commit Title: `release: v1.21.0`